Automattic/newspack-blocks
# Google Ad Manager block

### All Submissions:


### Changes proposed in this Pull Request:

Provides a Gutenberg block for inserting pre-defined ad inventory into the editor. Ads are pulled from the Newspack plugin where the Ads Wizard allows users to insert ad codes from Google Ad Manager.

### How to test the changes in this Pull Request:

1. Apply the patch
1. Add some ads using the Google Ad Manager wizard, with the following names and codes:
    1. Name: `Sidebar 300`, Code: `<amp-ad width=300 height=250
    type="doubleclick"
    data-slot="/21819266620/ll_sidebar_med_rect">
</amp-ad>`
    1. Name: `Leaderboard`, Code: `<amp-ad width=468 height=60
    type="doubleclick"
    data-slot="/21819266620/sdfsdf"
    data-multi-size="320x50">
</amp-ad>`
    1. Name: `Large Leaderboard`, Code: `<amp-ad width=728 height=90
    type="doubleclick"
    data-slot="/21819266620/fsadfsdf">
</amp-ad>`
1. Add a new post, and add an Ad Manager block
1. Observe the drop down box in the block settings for selecting an ad unit, which will contain the ad units you added through the wizard
1. Choose the ad units in turn and observe the block dynamically changing size
1. Choose the different block alignment options and verify the ad is placed as expected.
1. Save and view the post to verify the ad code is printed as expected

### Other information:
